the-collab-lab/tcl-70-smart-shopping-list
# Added styling and darkmode for forms

## Description

This code added css to forms including darkmode

## Related Issue

closes #87 

## Acceptance Criteria

- [x] Add css to forms

## Type of Changes

## Type of Changes
|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: Enhancement     |


## Updates

### Before

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/6857737b-23e5-49d5-9a5d-24918c757e4f)
![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/12186ca2-04b6-4229-b86d-98675f8d8129)

### After

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/c2d45947-d46d-4ebd-9afa-696ea642bee9)
![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/dd1042f8-140e-4e90-b3c3-d46fbeb1493c)

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/12508479-8647-46c7-96b5-e2c0d46c7f03)
![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/50a85747-407a-4fb7-b488-19c4c0416543)


## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin km-fix-forms` and check that branch out with `git checkout km-fix-forms` 
* Run `npm start`
* In the browser, go to  `http://localhost:3000/` to view the changes to the add list form on the homepage
* Click on a list to go to the list page and view the add item form
* For both lists, check if their darkmode versions 
* Test the list to see if they work in both  "light" and dark mode
